src/openmrs-react-root-decorator.js

```diff
@@ -41,7 +47,7 @@ export default function decorateOptions(opts) {
           });
         }
 
-        if (this.props.throwErrorsToConsole) {
+        if (opts.throwErrorsToConsole) {
```

Comment:
Since this means throwErrorsToConsole can no longer be passed as a prop, does this break the tests? Wanna update the tests to not pass it as a prop?

Reply:
👍 updated